### PR TITLE
Add some config files!

### PR DESCRIPTION
**Pull request by FredTheDino**

### CODE REVIEW

**Comment by FredTheDino** [comment]
Please do merge ASAP. And don't forget to publish it! :D

File: website/__init__.py [hunk 1]
```diff
@@ -150,11 +150,11 @@ def competitions_en():
 
 @app.route("/gamejam/se/")
 def gamejam_se():
-    return render_page("website/pages/gamejam_se.md", "/gamejam/", 4, True)
```
Comment (FredTheDino) [inline website/__init__.py:153]: `autocmd FileType rust,html,c,cpp,java,python autocmd BufWritePre <buffer> %s/\s\+$//e` @EMattfolk

File: website/pages/cheats_se.md [hunk 2]
```diff
@@ -6,3 +6,11 @@ dessa fuskpapper så säkert vi kan.
 Ett "enkelt" git fusk papper med dom mest använda
 kommandona, specialanpassat för Nollan. Tokbra, ju!
 
+## <a href="/vimrc" download=".vimrc">.vimrc</a>
+En bra första .vimrc, med lite extra python grejer inslängda!
+Komihåg att döpa om den till ".vimrc"!
```
Comment (sornas) [inline website/pages/cheats_se.md:11]:
```suggestion
Kom ihåg att döpa om den till ".vimrc"!
```

File: website/pages/cheats_se.md [hunk 3]
```diff
@@ -6,3 +6,11 @@ dessa fuskpapper så säkert vi kan.
...
+
+## <a href="/emacs_config" download=".emacs">.emacs</a>
+En konfiguration för Emacs som förenklar några av svårigheterna
+med att först börja använda Emacs. Värt att undersöka!
+Komihåg att döpa om den till ".emacs"!
```
Comment (sornas) [inline website/pages/cheats_se.md:16]:
```suggestion
Kom ihåg att döpa om den till ".emacs"!
```

File: website/other/vimrc [hunk 4]
```diff
@@ -0,0 +1,57 @@
+" A simple default configuration for beginner Vim usage.
+"
+" If you have any questions about this file, send and email to
```
Comment (sornas) [inline website/other/vimrc:3]:
```suggestion
" If you have any questions about this file, send an email to
```

File: website/other/emacs_config [hunk 5]
```diff
@@ -0,0 +1,10 @@
+;;Place this file your home directory (the command: "cd" will bring you there)
+
+;;If region is not active – no visible selection –
+;;then deafult shortcuts works as it as it does normally in Emacs
```
Comment (sornas) [inline website/other/emacs_config:4]:
```suggestion
;;then deafult shortcuts works as it does normally in Emacs
```

File: website/other/vimrc [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+" edvth289@student.liu.se, made with <3 by LiTHe kod.
+
+" This is strictly for Vim, not the ancient Vi.
+set nocompatible
+
+" Keep the undo history, lets you close the buffer and
+" still undo.
+set undofile
```
Comment (sornas) [inline website/other/vimrc:11]:
```suggestion
" Keep the undo history, lets you close the buffer and
" still undo.
if !isdirectory($HOME."/.vim")
    call mkdir($HOME."/.vim", "", 0770)
endif
if !isdirectory($HOME."/.vim/undo-dir")
    call mkdir($HOME."/.vim/undo-dir", "", 0700)
endif
set undodir=~/.vim/undo-dir
set undofile
```

https://vi.stackexchange.com/a/53

I don't know if this works on Windows, but I think it should.

**Review by sornas — COMMENTED** [review]
I can't really speak for the .emacs.

**Review by vollells — APPROVED** [review]
Looks Good

File: website/__init__.py [hunk 7]
```diff
@@ -239,6 +239,16 @@ def gitcheatsheet():
...
     return static_page("website/other/gitcheatsheet.html")
 
+@app.route("/vimrc")
+def vimrc():
+    """ Get the vimrc. """
+    return send_file("other/vimrc", attachment_filename=".vimrc", as_attachment=True)
+
+@app.route("/emacs_config")
+def emacs():
+    """ Get the vimrc. """
```
Comment (EMattfolk) [inline website/__init__.py:249]: Emacs config?

**Pull request merged**